cmd/controller/controller.go

```diff
@@ -311,12 +329,7 @@ func (c *command) start(ctx context.Context) error {
 	if c.EnableDynamicConfig {
 		configSource, err = clusterconfig.NewAPIConfigSource(adminClientFactory)
 	} else {
-		var clusterConfig *v1beta1.ClusterConfig
-		clusterConfig, err = config.LoadClusterConfig(c.K0sVars)
-		if err != nil {
-			return fmt.Errorf("failed to load cluster config: %w", err)
-		}
-		configSource, err = clusterconfig.NewStaticSource(clusterConfig)
+		configSource, err = clusterconfig.NewStaticSource(nodeConfig)
```

Comment:
hmm, I think we should use the full cluster config here

Reply:
NodeConfig is always a full cluster config when dynamic config is not enabled. It is only stripped down to BootstrapConfig when merged to a dynamic one.

Reconcile should only deal with ClusterWideConfig, any component that uses anything from the NodeConfig should  receive it as a static config in initialization, if it needs anything from the cluster wide config, it should be received from reconcile.

Reply:
Now `GetBootstrappingConfig()` is never called so it too can be removed.

Reply:
GetClusterWideConfig() is only used upon creating the initial clusterconfig api object.

Reply:
👍